src/components/app-demo/index.tsx

```diff
@@ -40,75 +31,33 @@ export class SmoothlyAppDemo {
 							action link
 						</smoothly-button>
 					</div>
-				</smoothly-room>
-				<smoothly-room path="input" label="Input">
+				</smoothly-app-room>
+				<smoothly-app-room path="/input" label="Input">
 					<smoothly-input-demo />
-				</smoothly-room>
-				<smoothly-room path="button" label="Button">
+				</smoothly-app-room>
+				<smoothly-app-room path="button" label="Button">
 					<smoothly-button-demo />
-				</smoothly-room>
-				<smoothly-room path="dialog" label="Dialog">
+				</smoothly-app-room>
+				<smoothly-app-room path="dialog" label="Dialog">
 					<smoothly-dialog-demo />
```

Comment:
why do some have / in front and some don't?

Reply:
The path is now written in the same way overall